### ISSUE TITLE
Video demo for README

### ISSUE THREAD

**Issue by Victor239** [sawhney17/logseq-citation-manager#48]
It would be very helpful to have a video demo to show how to quickly setup and use this plugin.

**Comment by kmk93posteo** [comment]
I agree, but for the time being, this 2-min preview gives an intro: https://twitter.com/aryansawhney17/status/1545715630392877062
